docs/installation.rst

```diff
@@ -1,19 +1,15 @@
 Installation Instructions
 =========================
 
-.. warning::
+Install via Anaconda
+--------------------
 
-  This package has a lingering dependency on an IRAF task that is accessed
-  through PyRAF.  IRAF/PyRAF can be obtained through
-  `Astroconda <http://astroconda.readthedocs.io/en/latest/>`_.  Though many
-  tasks will work without IRAF installed, some will fail under certain
-  circumstances.  This depencency will be removed in a future release as
-  resources permit.
+TBD.
```

Comment:
@sean-lockwood just want to confirm that the intent is to have these instructions not present in this release?

Reply:
I'm going to work on the conda installation after the merge.  Will update the instructions accordingly.